scio-parquet/src/main/java/com/spotify/scio/parquet/avro/LogicalTypeSupplier.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.parquet.avro;
+
+import org.apache.avro.Conversions;
+import org.apache.avro.data.TimeConversions;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.specific.SpecificData;
+import org.apache.parquet.avro.SpecificDataSupplier;
+
+/** A SpecificDataSupplier that supplies built-in conversions for Avro LogicalTypes. */
+public class LogicalTypeSupplier extends SpecificDataSupplier {
+  @Override
+  public GenericData get() {
+    return new SpecificData() {
+      {
+        addLogicalTypeConversion(new TimeConversions.DateConversion());
+        addLogicalTypeConversion(new TimeConversions.TimeConversion());
+        addLogicalTypeConversion(new TimeConversions.TimestampConversion());
+        addLogicalTypeConversion(new Conversions.DecimalConversion());
```

Comment:
Maybe it's worth to add all remaining conversions in `TimeConversions` and `Conversions`?

Reply:
oh, I guess I forgot a few `*Micros` conversions, and UUID.... will add them, good catch!

Reply:
added!